packages/fast-components/src/slider-label/fixtures/base.html

```diff
@@ -0,0 +1,8 @@
+<fast-design-system-provider use-defaults>
+    <!-- Basic -->
+    <div style="margin: 20px">
+        <fast-slider-label>
+            test
+        </fast-slider-label>
+    </div>
+</fast-design-system-provider>
```

Comment:
is this example exhaustive enough to ensure we don't regress the control in the future when making changes? Should we have examples passing position and hide-mark?

Reply:
Should this include vertical examples as well since you have that included?

Reply:
Seems like the slider-label is somewhat exhaustively tested in the slider fixtures. Does it make sense to do that again here? I can try and add a few things, for sure hide-mark.

Reply:
I think that's a fair point but it assumes that people will test across controls. I think we should include examples under the assumption that people sometimes develop in isolation and do not do what is right :).

Reply:
I added position, vertical and hide-mark examples